packages/bitcore-node/src/modules/ethereum/api/csp.ts

```diff
@@ -168,6 +170,12 @@ export class ETHStateProvider extends InternalStateProvider implements CSP.IChai
           .on('transactionHash', resolve)
           .on('error', reject)
           .catch(e => reject(e));
+        if (Etherscan.supportsNetwork(network)) {
```

Comment:
shouldnt these be wrapped in a catch?

Reply:
I'm not sure, does request throw? I didn't think it did